build/all/Dockerfile

```diff
@@ -55,6 +55,7 @@ ARG USER_GID=$USER_UID
 RUN groupadd --gid $USER_GID $USERNAME && \
   useradd --uid $USER_UID --gid $USER_GID -m $USERNAME
 USER nonroot:nonroot
+RUN apt-get update && apt-get install -y bash git
```

Comment:
should this be done before changing the user?

Reply:
You are correct!